go/client/chat_cli_utils.go

```diff
@@ -127,7 +127,7 @@ func parseConversationResolver(ctx *cli.Context, tlfName string) (resolver conve
 		return resolver, err
 	}
 	if resolver.TopicType == chat1.TopicType_CHAT && len(resolver.TopicName) != 0 {
-		return resolver, errors.New("multiple topic names are not yet supportted with chat conversations")
+		return resolver, errors.New("multiple topic names are not yet supported")
```

Comment:
Yup it does. Well what I had in mind is dev conversations can still have multiple topic per TLF but I guess for CLI interface, not mentioning dev conversations is fine!